openscap-policies/src/main/scala/com/normation/plugins/openscappolicies/extension/OpenScapNodeDetailsExtension.scala

```diff
@@ -0,0 +1,87 @@
+package com.normation.plugins.openscappolicies.extension
+
+import com.normation.plugins.openscappolicies.OpenScapReport
+import com.normation.plugins.openscappolicies.services.{OpenScapReportReader, ReportSanitizer}
+import com.normation.plugins.{PluginExtensionPoint, PluginStatus}
+import com.normation.rudder.web.components.ShowNodeDetailsFromNode
+import net.liftweb.common.{EmptyBox, Full, Loggable}
+
+import scala.reflect.ClassTag
+import scala.xml.NodeSeq
+import net.liftweb.util.CssSel
+import net.liftweb.util.Helpers._
+import com.normation.box._
+import com.normation.inventory.domain.NodeId
+
+class OpenScapNodeDetailsExtension(
+    val status: PluginStatus
+  , openScapReader: OpenScapReportReader
+  , reportSanitizer: ReportSanitizer)(implicit val ttag: ClassTag[ShowNodeDetailsFromNode]) extends PluginExtensionPoint[ShowNodeDetailsFromNode] with Loggable {
+
+  def pluginCompose(snippet: ShowNodeDetailsFromNode) : Map[String, NodeSeq => NodeSeq] = Map(
+      "popupDetails" -> addOpenScapReportTab(snippet) _
+    , "mainDetails"  -> addOpenScapReportTab(snippet) _
+  )
+
+  /**
+   * Add a tab:
+   * - add an li in ul with id=openScapExtensionTab
+   */
+  def addOpenScapReportTab(snippet: ShowNodeDetailsFromNode)(xml: NodeSeq): NodeSeq = {
+    val content = openScapReader.checkOpenScapReportExistence(snippet.nodeId) match {
+      case eb: EmptyBox =>
+        val e = eb ?~! "Can not display OpenScap report for that node"
+        (<div class="error">{e.messageChain}</div>)
+      case Full(existence) =>
+        existence match {
+          case false =>
+            <div id="openscap_reports">
+              <p>That tab gives access to OpenScap reports configured for that node</p>
+              <div class="error">No OpenScap report available for node {snippet.nodeId.value}</div>
+            </div>
+          case true =>
+            frameContent(snippet.nodeId)(openScapExtensionXml)
+        }
+    }
+
+
+      //tabContent(Map("key" ->"value"))(openScapExtensionXml)
+    val tabTitle = "OpenScap"
```

Comment:
open scap write it `OpenSCAP`, we should keep that typography for user facing things